kfdef/kfctl_anthos.yaml

```diff
@@ -0,0 +1,255 @@
+# This is the config to install Kubeflow on an existing k8s cluster.
+# If the cluster already has istio, comment out the istio install part below.
```

Comment:
I thought the assumption is that Anthos clusters already have istio?

Reply:
Ah, updated the comment (was copy from kfctl_istio)